site/content/includes/installation/nginx-plus/docker-registry-secret.md

```diff
@@ -0,0 +1,19 @@
+---
+docs: "DOCS-000"
+---
+
+{{<note>}}If you would rather pull the NGINX Plus image and push to a private registry, you can skip this specific step and instead follow [this step]({{<relref "installation/nginx-plus-jwt.md#pulling-an-image-for-local-use">}}).{{</note>}}
```

Comment:
Small nit:

But going off of this exact comment from @ADubhlaoich https://github.com/nginxinc/nginx-gateway-fabric/pull/2546#discussion_r1762915689, there are a few places below that I didn't mark that I will let you resolve.

```suggestion
{{< note >}} If you would rather pull the NGINX Plus image and push to a private registry, you can skip this specific step and instead follow [this step]({{<relref "installation/nginx-plus-jwt.md#pulling-an-image-for-local-use">}}). {{< /note >}}
```